src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -78,8 +78,32 @@ protected override string InternalGetProcessorArchitecture()
             return DesktopOsHelper.IsWindows() ? WindowsNativeMethods.GetProcessorArchitecture() : null;
         }
 
+        /// <summary>
+        /// The name of the operating system is important to the STS, as some CA policies 
+        /// will look at x-client-os; as such the name of the OS should be par-sable by the STS. 

```

Comment:
```suggestion
        /// will look at x-client-os; as such the name of the OS should be parseable by the STS. 
```

[nit] I don't think there should be a hyphen but if you want one then it should be parse-able.

Reply:
Thx, the VS spell check that I use suggested this and I blindly trusted it ...